templates/locales/en.yml

```diff
@@ -2192,6 +2192,8 @@ en:
         running_solo_again: "Running chef-solo again (failed to converge)..."
         running_zero: "Running chef-client (local-mode)..."
         running_zero_again: "Running chef-client (local-mode) again (failed to converge)..."
+        missing_chef_node_folder: |-
+          Nodes folder that Chef requires is missing in this Vagrant project
```

Comment:
I think this would read clearer if it included the missing folder in question. Then someone would be able to know which folder is missing. Something like:

```
nodes_path_missing: |-
  The following `nodes_path` is not present: %{path}
```

Then I'd move the error key to here, next to the other chef config error keys: https://github.com/mitchellh/vagrant/blob/master/templates/locales/en.yml#L1578

Then when you iterate over each possible `nodes_path` array in the `#validate` config method, you can raise the exception like so:

```
errors << I18n.t("vagrant.config.chef.nodes_path_missing", path: nodes_path)
```